en/developer_guide/install.rst

```diff
@@ -1,149 +1,111 @@
-Download OpenOrchestra
-======================
+Install OpenOrchestra on your computer
+======================================
 
 Open Orchestra is composed of two projects : *open-orchestra* which is the CMS Backoffice and *open-orchestra-front-demo* which
 is the Frontoffice part that will display the sites and pages created in the CMS.
 
-Requirements
-------------
+This tutorial has been tested on a debian 7.6.
 
-To install Open Orchestra you need to have installed the following software on your server:
+Install the environment
+-----------------------
 
-* Apache2 ``sudo aptitude install apache2``
-* PHP ``sudo aptitude install php5-dev php5-cli php-pear``
-* php-mongo  ``sudo pecl install mongo``
-* Activate php mongo extension copy ``extension=mongo.so`` on your php.ini
+Install the php server
+~~~~~~~~~~~~~~~~~~~~~~
 
-For developers, we provide a Vagrant-powered environment with provisionning so you get minimal setup actions to do.
-
-Install Vagrant
----------------
-The project is running on a Vagrant virtual environment built on VirtualBox to be production ready.
+Install php5 and the mongo extension :
 
 .. code-block:: bash
 
-    $ aptitude install virtualbox
-    $ wget https://dl.bintray.com/mitchellh/vagrant/vagrant_1.6.3_x86_64.deb
-    $ dpkg -i vagrant_1.6.3_x86_64.deb
-
-Install ansible
----------------
+    $ sudo aptitude install php5-cli php5-dev php-pear php5-curl
```

Comment:
As https://github.com/open-orchestra/open-orchestra-docs/pull/52/files shows, there are some other non-standard extensions required by Orchestra, maybe the doc could talk about them (Imagick, Apc, Mbstring)